### DISCUSSION TITLE
Starting a collection of themes

### DISCUSSION THREAD

**Discussion by leandwo** [qutebrowser/qutebrowser#6612] [Show and tell]
I'm creating a collection of themes, starting with onedark.

https://github.com/leosolid/qutebrowser-themes

PRs welcome. Let me know if there are certain themes you want ported.

**Comment by The-Compiler** [comment]
Mind opening a PR to add it to the [list of themes](https://github.com/qutebrowser/qutebrowser/blob/master/doc/help/configuring.asciidoc#pre-built-colorschemes) in the configuring docs?

**Reply by leandwo** [reply]
Done.
